client/blocks/followers-count/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+import { getSelectedSite } from 'state/ui/selectors';
+
+class FollowersCount extends Component {
+	render() {
+		const { site, translate } = this.props;
+
+		if ( ! site || ! site.subscribers_count ) {
+			return null;
+		}
+
+		return (
+			<div className="followers-count">
+				<Button borderless href={ '/people/followers/' + site.slug }>
+					{ translate( 'Followers' ) } <Count count={ site.subscribers_count } />
+				</Button>
+			</div>
+		);
+	}
+}
+
+export default connect( ( state ) => {
+	const site = getSelectedSite( state );
+
+	return {
+		site
+	};
+} )( localize( FollowersCount ) );
```

Comment:
fun fact!

we can use object shorthand notation here and remove the noise of the curlies and `return`

```js
export default connect( state => ( {
	site: getSelectedSite( state ),
} ) )( localize( FollowersCount );
```

notice too that I surrounded the object's curlies (`{}`) in parentheses. normally the fat-arrow function will simply return the value from evaluating the expression on the right side _if_ there are no `{}`s. `add = ( a, b ) => a + b`. In this case, since we want to return an object the syntax is ambiguous. ES2015 resolves this ambiguity with the parentheses wrapping the object. :smile: